docs/docs/cicd.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_label: CI/CD
+---
+
+# Using OPA in CI/CD Pipelines
```

Comment:
Should we perhaps spell out "CI/CD" once somewhere? Or add an abbrev tooltip? 🤔

Reply:
Previously, I've used this
```
The <abbr title="left-hand side">LHS</abbr> `input.users.name` is part of the _unknowns_ (`input.users`), so the expression contributes to our conditions:
```

Reply:
Thanks, that's nice. I have added that now